src/main/java/org/generationcp/middleware/service/api/study/germplasm/source/GermplasmStudySourceService.java

```diff
@@ -0,0 +1,18 @@
+package org.generationcp.middleware.service.api.study.germplasm.source;
+
+import java.util.List;
+
+public interface GermplasmStudySourceService {
+
+	List<GermplasmStudySourceDto> getGermplasmStudySourceList(final GermplasmStudySourceRequest searchParameters);
```

Comment:
I found the name of this function hard to read, I will rename it to getGermplasmStudySources as well as both counts functions given that we are not counting lists but germplasm study sources